sdk/messaging/eventgrid/aznamespaces/client_test.go

```diff
@@ -312,11 +312,17 @@ func TestSimpleErrors(t *testing.T) {
 	_, err := c.PublishCloudEvents(context.Background(), c.TestVars.Topic, []messaging.CloudEvent{
 		{},
 	}, nil)
-	var respErr *azcore.ResponseError
 
+	// this'll lead to an error message like this:
+	// "error": {
+	// 	"code":"BadRequest",
```

Comment:
Is there a stable error code we can check for?

Reply:
But error code?

Reply:
Added (and in another spot as well)